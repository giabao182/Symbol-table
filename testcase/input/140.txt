INSERT id number
ASSIGN id 10
BEGIN
INSERT id string
ASSIGN id 'abc'
LOOKUP id
END
ASSIGN id 'xyz'