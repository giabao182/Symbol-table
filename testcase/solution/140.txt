success
success
Redeclared: INSERT id string
TypeMismatch: ASSIGN id 'abc'
0
TypeMismatch: ASSIGN id 'xyz'