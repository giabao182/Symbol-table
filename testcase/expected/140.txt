success
success
success
success
1
TypeMismatch: ASSIGN id 'xyz'